client/filter.ts

```diff
@@ -0,0 +1,28 @@
+import { isArray } from "lodash";
+
+import { createLogger } from "../log";
+
+const logger = createLogger({ name: "filter" });
+
+// jq-node doesn't support `import` syntax
+const jq = require("node-jq");
+
+export const jpFilter = async (
+  data: any,
+  jqHeader: string | string[] | undefined
+): Promise<any> => {
+  const query = isArray(jqHeader) ? jqHeader[0] : jqHeader;
```

Comment:
Wouldn't you want to run this multiple times if there are multiple headers?
```suggestion
  const query = isArray(jqHeader) ? jqHeader : [jqHeader];
```

Reply:
You mean chain the header jq commands? The output of the first is the input of the second?
I don't know if that's helpful, I think chains should be expressible in one query.